### ISSUE TITLE
[runtime] Support if in delete statement

### ISSUE THREAD

**Issue by pingginp** [pingginp/cqlc#13]
https://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlDelete.html

````
DELETE FROM cycling.cyclist_name 
WHERE id =e7ae5cf3-d358-4d99-b900-85902fda9bb0 
if firstname='Alex' and lastname='Smith';
````

**Comment by pingginp** [comment]
there is another issue `Eq` is only supported on partition columns, for non partition columns, it is not supported, so the generator needs to be changed as well

A quick hack would be allow eq on columns, which is actually a valid case to support `IF` statements ...

**Comment by pingginp** [comment]
fixed in #14
